plugins/main/public/components/agents/fim/inventory/registry-table.tsx

```diff
@@ -11,36 +11,26 @@
  */
 
 import React, { Component } from 'react';
-import {
-  EuiFlexGroup,
-  EuiFlexItem,
-  EuiBasicTable,
-  Direction,
-} from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiIconTip } from '@elastic/eui';
 import { WzRequest } from '../../../../react-services/wz-request';
 import { FlyoutDetail } from './flyout';
-import { filtersToObject } from '../../../wz-search-bar';
 import { formatUIDate } from '../../../../react-services/time-service';
-import {
-  UI_ERROR_SEVERITIES,
-  UIErrorLog,
-  UIErrorSeverity,
-  UILogLevel,
-} from '../../../../react-services/error-orchestrator/types';
-import { UI_LOGGER_LEVELS } from '../../../../../common/constants';
-import { getErrorOrchestrator } from '../../../../react-services/common-services';
+import { TableWzAPI } from '../../../common/tables';
+import { SEARCH_BAR_WQL_VALUE_SUGGESTIONS_COUNT } from '../../../../../common/constants';
+
+const searchBarWQLOptions = {
+  implicitQuery: {
+    query: 'type=registry_file',
+    conjunction: ';',
+  },
+};
+
+const searchBarWQLFilters = { default: { q: 'type=registry_file' } };
```

Comment:
![image](https://github.com/wazuh/wazuh-kibana-app/assets/42900763/7f680eb4-a2b9-49d9-9133-4885629d668a)
![image](https://github.com/wazuh/wazuh-kibana-app/assets/42900763/fbfbfe8b-7d9d-445c-bc6b-cf9b74d51de7)

As we are not getting any results on the table itself, we verified the main query and tested in the Dev Tools. We believe that the correct query should be `registry_key` instead of `registry_file`

Reply:
Fixed in https://github.com/wazuh/wazuh-kibana-app/pull/5444/commits/d8c4ab5c8c62c98fa7f593c01700fe78e5947ec1.